### ISSUE TITLE
Call back?

### ISSUE THREAD

**Issue by uxlayouts** [dbramwell/react-animate-on-scroll#11]
This plugin is amazing! Thank you so much for making it. Sometimes I need to setup a GSAP timeline with staggered animations etc. Is there any way you can add a simple call back event that fires once so I can run my timeline for more complex animations?

**Comment by dbramwell** [comment]
Sorry for the late reply to this. I guess we could do a sort of callback. Just executing a function in a timeout that waits for delay + duration seconds? If the animation is cancelled by scrolling away from the element then I guess you'd want the callback to not be executed?

Would you want something like this:
```
<ScrollAnimation onComplete={function() {alert("complete")}}>
```

Then around [here](https://github.com/dbramwell/react-animate-on-scroll/blob/master/src/scroll-animation.js#L62) we could add something along the lines of:
```
if (this.props.onComplete) {
  var callback = setTimeout(this.props.onComplete, this.props.delay + this.props.duration);
  timeouts.push(callback);
}
```

**Comment by uxlayouts** [comment]
Nicee Thank you!

**Comment by dbramwell** [comment]
It's not implemented yet. Leaving this open for now as a reminder

**Comment by dbramwell** [comment]
I've given it a go. Available in v1.5.1.

There's an example [here](https://dbramwell.github.io/react-animate-on-scroll/#onComplete)

Give it a try and let me know if there's any issues.
